source/javascript/src/daily.js

```diff
@@ -443,7 +445,15 @@ document.addEventListener('DOMContentLoaded', (event) => {
   time.addEventListener('change', () => {
     date.style.visibility = 'visible'
   })
-  getLogInfoAsJSON(populateDailyLog)
+
+  fetch('https://api.quotable.io/random')
```

Comment:
Quotes FTW 😄

Reply:
Good catch; We should probably add the new ReST API calls to our architecture diagrams as well. @yuzilyu what do you think?

Reply:
Sounds good!